### DISCUSSION TITLE
Problem seeing data in production but I can see it on localhost, using the same branch

### DISCUSSION THREAD

**Discussion by saulsolorzano** [planetscale/discussion#419] [Q&A]
Hi!

First time using planetscale, so far it's been awesome. The only problem I'm facing is that, I currently only have one branch, the main one. I did the original schema and then I had to do a branch to create a new field. I merged that branch to main and I can see all my data if i'm in my local environment. As well as using the console for that branch. If I use select * from Product; I can see everything there.

The problem is that I don't see the new field on production. The site is hosted on vercel and I'm using the correct env variable. I have tried looking at the docs but can't really find anything related to this. Have any of you experienced something like this before?

I'm also using prisma as an ORM 

Thanks

**Comment by jgreet** [comment]
Hi,

If you're still stuck on this problem it might be best to [open a support ticket ](https://support.planetscale.com/hc/en-us/requests/new) with some more details like the database name, the exact errors you're seeing in production and a copy of your Prisma schema file.

Typically if someone can't see the latest database schema from one particular environment, that environment will turn out to be  connected to the wrong branch. But you mentioned you only have the one branch, so perhaps the Prisma schema is out of date in the Vercel deploy? I don't think we can figure out what's going wrong without knowing more specifics.

**Reply by saulsolorzano** [reply]
Hey @jgreet thank you so much! I'll do that!
